zambeze/orchestration/message/abstract_message.py

```diff
@@ -7,4 +8,4 @@ class AbstractMessage(ABC):
 
     type: MessageType
     """Returns the Message as a dict"""
-    data: dict
+    data: Any
```

Comment:
Curious what the rationale is in changing data from 'dict' to 'any' type. I think messages, by default, will be parseable as dictionary-style structures... e.g., msg.data.body.xxx.yyy....

Reply:
This is because the type checker complains otherwise. I.e. if I have

```Python
msg: AbstractMessage = input_message

if msg.data.type == MessageType.ACTIVITY:
  pass
```

The typechecker will throw an error if you use dict.